gensrc/java/com/starrocks/udf/UDFClassAnalyzer.java

```diff
@@ -0,0 +1,71 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021 StarRocks Limited.
+
+package com.starrocks.udf;
+
+import java.lang.reflect.Method;
+import java.lang.reflect.Modifier;
+import java.lang.reflect.Type;
+import java.util.HashMap;
+import java.util.Map;
+
+public class UDFClassAnalyzer {
+    static Map<String, String> anlyMap = new HashMap<>();
+
+    static {
+        anlyMap.put("boolean", "Z");
+        anlyMap.put("byte", "B");
+        anlyMap.put("char", "C");
+        anlyMap.put("short", "S");
+        anlyMap.put("int", "I");
+        anlyMap.put("long", "J");
+        anlyMap.put("float", "F");
+        anlyMap.put("double", "D");
+        anlyMap.put("void", "V");
+    }
+
+    private static String getSign(String typeName) {
```

Comment:
Is there already a lib function do this thing?

Reply:
We need to get a correct function signature to make function calls. This is very useful. We need to distinguish between boxed classes and primitive classes